app/src/test/java/com/example/molliestephenson/tictactoe/MobilePlayerTest.java

```diff
@@ -0,0 +1,29 @@
+package com.example.molliestephenson.tictactoe;
+
+import org.junit.Test;
+
+import ttt.game.Board;
+import ttt.game.Marks;
+
+import static org.junit.Assert.assertEquals;
+
+public class MobilePlayerTest {
+
+   private MobilePlayer player = new MobilePlayer(Marks.X);
+
+    @Test
+    public void knowsItsMark() {
+        assertEquals(Marks.X, player.getMark());
+    }
+
+    @Test
+    public void returns0ForLocation() throws Exception {
+        Board board = new Board(3);
+        assertEquals(0, player.getLocation(board));
+    }
+
+    @Test
+    public void testPlayerType() throws Exception {
+        assertEquals(MobilePlayer.class, player.playerType());
```

Comment:
This test is a bit of a code smell: Knowing the class of a thing has nothing to do with 'TTT' , 'Players' or 'MobilePlayers'...
What is the value of the method and by extension the test?